src/postgres/commands/configurePostgresFirewall.ts

```diff
@@ -16,7 +17,9 @@ import { PostgresServerTreeItem } from "../tree/PostgresServerTreeItem";
 
 export async function configurePostgresFirewall(context: IActionContext, treeItem?: PostgresServerTreeItem): Promise<void> {
     if (!treeItem) {
-        treeItem = <PostgresServerTreeItem>await ext.rgApi.appResourceTree.showTreeItemPicker(PostgresServerTreeItem.contextValue, context);
+        treeItem = await ext.rgApi.pickAppResource<PostgresServerTreeItem>(context, {
+            filter: [postgresSingleFilter, postgresFlexibleFilter]
```

Comment:
Just double checking again, `PostgresServerTreeItem.contextValue` is intentionally dropped?

Reply:
Also this same pattern appears in a couple of the other commands.

Reply:
Yeah, the way this works is the filter will give you the `AppResource` that lives on `SubscriptionTreeItem`.  The `expectedContextValue` is only used for searching for children on that `AppResource` so if it's omitted, then it's assumed that the picker ends at the `AppResource`

Reply:
Yes, the context value is only used for selecting children of top level app resources tree items (AppResource).